samples/camera/src/ucam3.h

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2019 Centaur Analytics, Inc.
```

Comment:
Is this [Centaur Analytics, Inc](https://www.linkedin.com/company/centaur-analytics-inc./)? Do we have the proper licensing rights to use this code?

Reply:
Sorry for the big delay. This is the company. However I own this code, so it's now under my name.